docs/ide/using-regular-expressions-in-visual-studio.md

```diff
@@ -53,7 +53,7 @@ Here are some examples:
 |Escape the character following the backslash| \\ |`\^` matches the character ^.|
 |Specify the number of occurrences of the preceding character or group|{x}, where x is the number of occurrences|`x(ab){2}x` matches "xababx", and `x(ab){2,3}x` matches "xababx" and "xabababx" but not "xababababx".|
 |Match text in a Unicode character class, where "X" is the Unicode number. For more information about Unicode character classes, see<br /><br /> [Unicode Standard 5.2 Character Properties](http://www.unicode.org/versions/Unicode5.2.0/ch04.pdf).|\p{X}|`\p{Lu}` matches "T" and "D" in "Thomas Doe".|
-|Match a word boundary|`\b` (Outside a character class \b specifies a word boundary, and inside a character class specifies a backspace).|`\bin` matches "in" in "inside" but not "pinto".|
+|Match a word boundary (Outside a character class specifies a word boundary, and inside a character class specifies a backspace).|\b|`\bin` matches "in" in "inside" but not "pinto".|
```

Comment:
@lightcici Thanks for contributing! I like the removal of the backticks around \b, but I think the description of what "\b" does belongs in the middle column, not the first column, because it's a description of \b, not a description of matching a word boundary. Does that make sense to you too?

Reply:
@gewarren Thanks for your reply. Yes, this makes sence to me. I am just thinking to align the format for all the rows. For example for `\p{X}` above this, do you think it's a good idea to put `where "X" is the Unicode number` to the middle column rather than the first column, because it's a description for `X` ?

Reply:
@lightcici Yes, I do think the description for X should be moved to the center column. Can you make those changes? Thank you for helping to improve the docs!